dissect/target/helpers/fsutil.py

```diff
@@ -16,9 +16,16 @@
 try:
     import bz2
 
-    HAVE_BZ2 = True
+    HAS_BZ2 = True
```

Comment:
As much as I like the more correct grammar, we currently use `HAVE_` in a ton of places in Dissect, so it's best to stick with it for now. We could change it at some point, but I'd prefer to change it all at once. And then this PR is not the place for that.